tue_robocup/package.xml

```diff
@@ -17,21 +17,17 @@
   <run_depend>challenge_final</run_depend>
   <run_depend>challenge_find_my_mates</run_depend>
   <run_depend>challenge_gpsr</run_depend>
+  <run_depend>challenge_hand_me_that</run_depend>
   <run_depend>challenge_help_me_carry</run_depend>
-  <run_depend>challenge_manipulation</run_depend>
```

Comment:
Why remove these?

Reply:
As these are not in the rulebook anymore, so why should these be installed?